client-grpc/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+edition = "2021"
+name    = "svc-scheduler-client-grpc"
+version = "0.2.5-develop.3"
```

Comment:
Please reset the version to "0.1.0" so the release workflow won't run into issues and we get a proper CHANGELOG generated